lib/extension.js

```diff
@@ -23,13 +24,31 @@ ZeebeModdleExtension.$inject = [ 'eventBus' ];
 
 ZeebeModdleExtension.prototype.canCopyProperty = function(property, parent) {
 
-  // check if property is allowed in parent
+  // (1) check if property is allowed in parent
   if (isObject(property) && !isAllowedInParent(property, parent)) {
+    return false;
+  }
 
+  // (2) check for specific scenarios
+  if (!this.canHostServiceTaskLikeProperties(property, parent)) {
     return false;
   }
 };
 
+ZeebeModdleExtension.prototype.canHostServiceTaskLikeProperties = function(property, parent) {
+
+  if (isZeebeServiceTaskSpecific(property)) {
+
+    var serviceTaskLike = getParent(parent, 'bpmn:IntermediateThrowEvent') || getParent(parent, 'bpmn:EndEvent');
+
+    if (serviceTaskLike) {
+      return isMessageEvent(serviceTaskLike);
+    }
+  }
+
+  return true;
```

Comment:
Usually we don't return in `canCopyProperty` => also see how our tests check for `not.to.be.false`. Hence I think we should not change that for consistency reasons (unless there is a particular reason?)

Reply:
The sample applies to https://github.com/zeebe-io/zeebe-bpmn-moddle/pull/15/commits/61822ec0afa91ad0d842c14d5925966d94783dbc#diff-63d870fcd395c9d6f4bd53f3f4bd7a29eb165aff442eacfbe955bfdbb96f1e9eR45

Reply:
For this I followed the behaviour I saw on the `canCopyProperty` in the camunda-bpmn-moddle repository (see screenshots for `canHostConnector` example).

<img width="754" alt="Screenshot 2021-09-22 at 09 14 54" src="https://user-images.githubusercontent.com/25825387/134299207-bd6de1bf-9516-4a2f-9b6c-47b4031af30b.png">
<img width="531" alt="Screenshot 2021-09-21 at 20 00 06" src="https://user-images.githubusercontent.com/25825387/134299255-3095d355-c801-413e-9ad1-b01083831c21.png">

Reply:
Ok, then we are good :-)